Source/WebCore/dom/Document.cpp

```diff
@@ -3011,6 +3012,17 @@ void Document::collectRangeDataFromRegister(Vector<WeakPtr<HighlightRangeData>>&
     }
 }
 
+static void repaintingRange(const AbstractRange& range)
```

Comment:
Nit - this should be a verb like `repaintRange(const AbstractRange& range)` (or even just `repaint(const AbstractRange& range)`).